CONTRIBUTING.md

```diff
@@ -113,6 +113,32 @@ If you'd like to contribute example `.bicep` files that showcase abilities of th
 
 **Note:** If you have never submitted a Pull Request or used git before, reading through the [Git tutorial](https://github.com/Azure/azure-quickstart-templates/blob/master/1-CONTRIBUTION-GUIDE/git-tutorial.md) in the azure-quickstart-template repo is a good place to start.
 
+### Snippets
+
+If you'd like to contribute to the collection of snippets:  
+
+* A snippet should be a single, generic, resource. Ensure your snippet meets this criteria.
+* Add a Bicep file here, using the naming convention `res-RESOURCENAME.bicep`: [`./src/Bicep.LangServer/Snippets/Templates`](./src/Bicep.LangServer/Snippets/Templates)
```

Comment:
Can we restructure this sentence to:

Add a Bicep file to [`./src/Bicep.LangServer/Snippets/Templates`](./src/Bicep.LangServer/Snippets/Templates) using the naming convention `res-RESOURCENAME.bicep`: